src/morale.cpp

```diff
@@ -0,0 +1,143 @@
+#include "morale.h"
+#include "itype.h"
+#include "cata_utility.h"
+#include "debug.h"
+#include "output.h"
+
+namespace {
+    static const std::string item_name_placeholder = "%i"; // Used to address an item name
+
+    const std::string &get_morale_data( const morale_type id )
+    {
+        static const std::array<std::string, NUM_MORALE_TYPES> morale_data = { {
+            { "This is a bug (player.cpp:moraledata)" },
+            { _( "Enjoyed %i" ) },
+            { _( "Enjoyed a hot meal" ) },
+            { _( "Music" ) },
+            { _( "Enjoyed honey" ) },
```

Comment:
@codemime compare capitalization in spelling of phrases ```Enjoyed honey``` and ```Ate Human Flesh```(Kevin's proposition sounds better than way it's implemented now).